### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by bamsi**
In this PR, I have made some changes in my mobile version to be responsive to the desktop version. The portfolio changes in respective to devise screen size.  I am requesting a code review of the portfolio desktop version.
Thanks

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -94,33 +171,47 @@ h2 {
...
+@media only screen and (min-width: 1200px) {
+  .work-list {
+    grid-template-columns: repeat(3, 369px);
+    justify-content: space-between;
+    column-gap: 24px;
+    row-gap: 32px;
+  }
+}
+
+.list-item {
```
Comment (Amrhub) [inline css/style.css:214]: Great implementation and usage of breakpoints in CSS 👏🏻 
- [ ] Let's kindly make sure our cards have a space at the bottom to match Figma design

Your implementation | Figma Design
:------:|:-----:
image | image

File: css/style.css [hunk 2]
```diff
@@ -8,73 +8,150 @@ html {
...
+ul {
+  list-style-type: none;
+}
+
 .main-container {
-  font-family: Inter, sans-serif;
   display: flex;
   flex-direction: column;
   align-items: center;
+  font-family: Inter, sans-serif;
```
Comment (Amrhub) [inline css/style.css:31]: - let's make sure we get rid of unwanted space at the footer to match the Figma design

Your implementation | Figma Design
:------:|:-----:
image | image

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @bamsi,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 2 above)
Comment (bamsi) [reply css/style.css:31]: Thanks for the comments, I have fixed all the issues.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @bamsi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**